conda_forge_admin_requests/core.py

```diff
@@ -0,0 +1,71 @@
+import subprocess
+import os
+import tempfile
+import requests
+import copy
+from conda_smithy.github import Github
+import csv
+
+
+def _get_core(url):
+    res = requests.get(url)
+    data = csv.reader(res.content.decode("utf-8").splitlines(), delimiter=',')
```

Comment:
This can be [`.text`](https://requests.readthedocs.io/en/latest/user/quickstart/#response-content), I think?

```suggestion
    data = csv.reader(res.text.splitlines(), delimiter=',')
```